### ISSUE TITLE
distclu/paraclu crashes if I don't set rowRanges(CTSStagCountSE(ce))$filteredCTSSidx <- Rle(TRUE)

### ISSUE THREAD

**Issue by a-klarkowska** [charles-plessy/CAGEr#120]
as per our discussion earlier

**Comment by charles-plessy** [comment]
Thanks!

Minimal reproducible example:

```
> rowRanges(CTSStagCountSE(exampleCAGEexp))$filteredCTSSidx <- NULL
> distclu(exampleCAGEexp)
Clustering...
	-> Zf.unfertilized.egg

 Error in cluster_start_idx + local_max_idx : 
  non-numeric argument to binary operator
```

**Comment by charles-plessy** [comment]
In the meantime, to generate a `filterLowExpCTSS` column you can also use the `filterLowExpCTSS()` function.  Set `threshold = 0` to not filter at all.

**Comment by charles-plessy** [comment]
Fixed in 2.11.4
